common.yml

```diff
@@ -0,0 +1,97 @@
+applications:
+  - prometheus
+
+parameters:
+  pkg.monitoring:
+    images:
+      thanos:
+        registry: quay.io
+        repository: thanos/thanos
+        tag: v0.24.0
+    prometheus:
+      ingress:
+        host: prometheus.example.com
+        annotations:
+          cert-manager.io/cluster-issuer: letsencrypt-production
+      proxy:
+        provider: keycloak-oidc
+        provider-display-name: "Test Account"
+        client-id: syn-monitoring-${cluster:name}
```

Comment:
This will most likely change. We should speak with Dani before inventing client names.

As a default ok.

Reply:
Fair enough. I think he likes underscores? :sweat_smile: 
We definitely should talk to him about naming as soon as we put this in production. I just chose some arbitrary default client id for now.